packages/combo-box/src/vaadin-combo-box-overlay.js

```diff
@@ -52,6 +52,12 @@ export class ComboBoxOverlay extends ComboBoxOverlayMixin(Overlay) {
 
     return memoizedTemplate;
   }
+
+  constructor() {
+    super();
+
+    this.requiredVerticalSpace = 200;
```

Comment:
I wonder if 200px is a good default value for combo-box?

Reply:
Should be a good start. We can consider changing it, or making this configurable if this doesn't fit every use case.

Reply:
We used to have a [hardcoded number](https://github.com/vaadin/vaadin-combo-box/blob/06865cc0ac2518dedaac37d49f3a331683a7d630/src/vaadin-combo-box-dropdown-wrapper.html#L327) in V14 - IIRC, it originates from the 1.x Paper elements based version. It makes no sense already in V10 where we switched to Lumo, but apparently no one noticed it 🙂  

Personally, I feel like 200px should be a reasonable default for the minumal overlay height.